Makefile

```diff
@@ -1,64 +1,58 @@
 contentpack: pex
 	mkdir -p out/
-	PEX_MODULE=contentpacks ./makecontentpacks ka-lite en 0.16 --out=out/en.zip --no-subtitles
+	PEX_MODULE=contentpacks ./makecontentpacks ka-lite en 0.17 --out=out/en.zip
 	./makecontentpacks minimize-content-pack.py out/en.zip out/en-minimal.zip
 	./makecontentpacks extract_khan_assessment.py out/en.zip
 	./makecontentpacks collectmetadata.py out/ --out=out/all_metadata.json
 
-es: pex
-	PEX_MODULE=contentpacks ./makecontentpacks ka-lite es 0.16 --out=out/langpacks/es.zip --no-assessment-resources --subtitlelang=es --interfacelang=es-ES --contentlang=es-ES
-
-
-pt-BR: pex
-	PEX_MODULE=contentpacks ./makecontentpacks ka-lite pt-BR 0.16 --out=out/langpacks/pt-BR.zip --no-assessment-resources --videolang=pt-BR --contentlang=pt-BR
-
-
-sw: pex
-	PEX_MODULE=contentpacks ./makecontentpacks ka-lite sw 0.16 --out=out/langpacks/sw.zip --no-assessment-resources --videolang=sw --subtitlelang=sw
-
-
-pt-PT: pex
-	PEX_MODULE=contentpacks ./makecontentpacks ka-lite pt-PT 0.16 --out=out/langpacks/pt-PT.zip --no-assessment-resources --videolang=pt-PT --contentlang=pt-PT
+ar: pex
+	PEX_MODULE=contentpacks ./makecontentpacks ka-lite ar 0.17 --out=out/langpacks/ar.zip --no-assessment-resource
 
+bg: pex
+	PEX_MODULE=contentpacks ./makecontentpacks ka-lite bg 0.17 --out=out/langpacks/bg.zip --no-assessment-resources
 
 bn: pex
-	PEX_MODULE=contentpacks ./makecontentpacks ka-lite bn 0.16 --out=out/langpacks/bn.zip --no-assessment-resources
+	PEX_MODULE=contentpacks ./makecontentpacks ka-lite bn 0.17 --out=out/langpacks/bn.zip --no-assessment-resources
 
+da: pex
+	PEX_MODULE=contentpacks ./makecontentpacks ka-lite da 0.17 --out=out/langpacks/da.zip --no-assessment-resources
 
 de: pex
-	PEX_MODULE=contentpacks ./makecontentpacks ka-lite de 0.16 --out=out/langpacks/de.zip --no-assessment-resources
+	PEX_MODULE=contentpacks ./makecontentpacks ka-lite de 0.17 --out=out/langpacks/de.zip --no-assessment-resources
 
+es: pex
+	PEX_MODULE=contentpacks ./makecontentpacks ka-lite es 0.17 --out=out/langpacks/es.zip --no-assessment-resources --subtitlelang=es --interfacelang=es-ES --contentlang=es-ES
 
 fr: pex
-	PEX_MODULE=contentpacks ./makecontentpacks ka-lite fr 0.16 --out=out/langpacks/fr.zip --no-assessment-resources
-
-
-da: pex
-	PEX_MODULE=contentpacks ./makecontentpacks ka-lite da 0.16 --out=out/langpacks/da.zip --no-assessment-resources
-
+	PEX_MODULE=contentpacks ./makecontentpacks ka-lite fr 0.17 --out=out/langpacks/fr.zip
```

Comment:
Why does fr.zip not have `--no-assessment-resources`? Looking at file sizes, it seems that it's around the same size as the Spanish (~88 MB)